build_tools/github_actions/configure_ci.py

```diff
@@ -224,6 +224,32 @@ def check_description_and_show_diff(
     )
 
 
+def parse_trailer_map_from_description(description: str):
+    trailer_lines = subprocess.run(
+        ["git", "interpret-trailers", "--parse", "--no-divider"],
+        input=description,
+        stdout=subprocess.PIPE,
+        check=True,
+        text=True,
+        timeout=60,
+    ).stdout.splitlines()
+
+    # Skip over multi-line or malformed trailers we don't want to support.
+    # https://github.com/iree-org/iree/issues/19240
+    # https://stackoverflow.com/q/66215644
+    # We could also handle multi-line git trailers, but we'd need to rework the
+    # .splitlines() call above.
+    trailer_lines = [line for line in trailer_lines if not line.startswith((" ", "\t"))]
+    trailer_lines = [line for line in trailer_lines if ":" in line]
```

Comment:
Does this need to also require that the `:` occurs in the middle of the string, not at the start or at the end as in `Notes:` ? I thought that played a role in `line.split(":")` returning a single element and failing the requirement of the structured binding to a pair `k, v` ?

Reply:
The lines there are doing the same thing. I added both for some safety / redundancy in case I'm missing some edge case.

Some printf debugging of the test cases:

### test_parse_trailer_map_multiline_trailer

```
First line

key: value line 1
  value line 2
```

Parsed by `git interpret-trailers` --> `.splitlines()` as
```
['key: value line 1', ' value line 2']
```

Either list comprehension will remove the second element of that list, which fails at the split -> unpack code.

### test_parse_trailer_map_multiline_trailer_skip_first

```
First line

key:
  value line 2
  value line 3
```

Parsed by `git interpret-trailers` --> `.splitlines()` as

```
['key: value line 2', ' value line 3']
```